workflows/wf_merlin_magic.wdl

```diff
@@ -47,11 +47,11 @@ workflow merlin_magic {
     Boolean call_shigeifinder_reads_input = false
     Boolean tbprofiler_additional_outputs = false
     String output_seq_method_type = "WGS"
-    Int srst2_min_cov = 90
-    Int srst2_max_divergence = 10
+    Int srst2_min_cov = 80
+    Int srst2_max_divergence = 20
     Int srst2_min_depth = 5
     Int srst2_min_edge_depth = 2
-    Int srst2_gene_max_mismatch = 10
+    Int srst2_gene_max_mismatch = 200
```

Comment:
Maybe it should be increased further

Reply:
@emmadoughty

Reply:
Resloved?

Reply:
Seems that the validation runs linked above don't have these values set by merlin magic--which leads me to assume that they're just using[ default srst2 values (i.e. 10)](https://github.com/katholt/srst2#current-release---v020---july-28-2016).